plugins/extra/remind.go

```diff
@@ -200,14 +212,89 @@ func (p *reminderPlugin) InitialDispatch(b *seabird.Bot, m *irc.Message) {
 	go p.remindLoop(b)
 }
 
+// Parses the text string and turns it into a time.Duration
+func (p *reminderPlugin) ParseTime(timeStr string) (time.Duration, error) {
+	buf := bytes.NewBufferString(timeStr)
+	parsed := &timeData{}
+
+	var tmp string
```

Comment:
This seems a bit more than is necessary--we can probably lean a lot on regexp here with something like https://play.golang.org/p/KzW4wrf1kj instead.